pkg/core/resources/store/store.go

```diff
@@ -124,6 +124,10 @@ func ErrorResourceConflict(rt model.ResourceType, name, mesh string) error {
 	return fmt.Errorf("Resource conflict: type=%q name=%q mesh=%q", rt, name, mesh)
 }
 
+func IsResourceConflict(err error) bool {
+	return err != nil && strings.HasPrefix(err.Error(), "Resource conflict")
+}
```

Comment:
merge origin/master, this function should be there

Reply:
Done